### PR TITLE
feat(scheduledItem): add scheduledSurfaceGuid as a field to ScheduledCuratedCorpusItem

### PR DESCRIPTION
**Pull request by sri-kirsh**
## Goal

- add `scheduledSurfaceGuid` as a required field to the entity `ScheduledCuratedCorpusItem`

Ticket: https://getpocket.atlassian.net/browse/INFRA-337

### CODE REVIEW

File: schema-admin.graphql [hunk 1]
```diff
@@ -345,6 +345,10 @@ type ScheduledCuratedCorpusItem {
     The associated Approved Item.
     """
     approvedItem: ApprovedCuratedCorpusItem!
+    """
+    Optional, The GUID of this schedudedSurface to which this item is scheduled. Example: 'NEW_TAB_EN_US'.
```
Comment (jpetto) [inline schema-admin.graphql:349]: we should be able to make this required - it's required in the db schema.

File: schema-admin.graphql [hunk 2]
```diff
@@ -345,6 +345,10 @@ type ScheduledCuratedCorpusItem {
...
+    """
+    scheduledSurfaceGuid: ID
```
Comment (nina-py) [inline schema-admin.graphql:351]: We'll never have a scheduled item without a surface it was scheduled on, so I think it's safe to make this prop required

File: src/admin/resolvers/mutations/ApprovedItem.integration.ts [hunk 3]
```diff
@@ -8,24 +8,24 @@ import {
   createApprovedItemHelper,
   createRejectedCuratedCorpusItemHelper,
   createScheduledItemHelper,
-  getServerWithMockedHeaders,
+  getServerWithMockedHeaders
```
Comment (sri-kirsh) [inline src/admin/resolvers/mutations/ApprovedItem.integration.ts:11]: lot of prettier changes in this file

File: src/admin/resolvers/mutations/ApprovedItem.integration.ts [hunk 4]
```diff
@@ -973,6 +973,10 @@ describe('mutations: ApprovedItem', () => {
       );
       expect(approvedItem.url).to.equal(input.url);
       expect(approvedItem.imageUrl).to.match(urlPattern);
+      expect(approvedItem.externalId).is.not.undefined;
```
Comment (sri-kirsh) [inline src/admin/resolvers/mutations/ApprovedItem.integration.ts:976]: this is the actual change, added assertions for fields we expect in the backfill code

**Review by nina-py — APPROVED** [review]

**Pull request merged**